pkg/testutil/testutil.go

```diff
@@ -34,12 +34,17 @@ type CheckFunc func(c *check.C) bool
 
 // WaitUntil repeatedly evaluates f() for a period of time, util it returns true.
 func WaitUntil(c *check.C, f CheckFunc) {
```

Comment:
Maybe we can add an option for it and we don't need two functions.

Reply:
I think the option is not necessary here.

Reply:
We don't need to add WaitUntilByInterval since these two functions do almost the same things

Reply:
updated.